packages/ember-testing/tests/reexports_test.js

```diff
@@ -19,7 +20,15 @@ class ReexportsTestCase extends AbstractTestCase {}
     exportName = path;
   }
 
-  ReexportsTestCase.prototype[`@test Ember.${path} exports correctly`] = function (assert) {
+  ReexportsTestCase.prototype[
+    `${testUnless(
+      DEPRECATIONS.DEPRECATE_IMPORT_EMBER(path).isRemoved
+    )} Ember.${path} exports correctly`
+  ] = function (assert) {
+    expectDeprecation(
+      /'ember' barrel file is deprecated/,
+      DEPRECATIONS.DEPRECATE_IMPORT_EMBER(path || exportName).isEnabled
```

Comment:
You should be able to use just `exportName`, no?

Reply:
nay, exportName is often `default` if it's a re-export form another module

Reply:
![image](https://github.com/emberjs/ember.js/assets/199018/7bfa8220-c385-40af-8f05-ec19591b21f6)

Reply:
Ah I see .. I was thrown by lines 19...21 above